### PR TITLE
Add Burp Suite 1.7.06

### PR DESCRIPTION
**Pull request by mcmahoniel**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [X] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/burp-suite.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  version '1.7.06'
+  sha256 'ce2da473fdb65f4704ad6597dcd6615ec84e7a4c3c81deaf4f2de360d362a9bd'
+
+  url "https://portswigger.net/Burp/Releases/Download?productId=100&version=#{version.major_minor_patch}&type=MacOsx"
+  name 'Burp Suite'
+  homepage 'https://portswigger.net/burp/'
+
+  installer manual: 'Burp Suite Free Edition Installer.app'
+
+  uninstall delete: '/Applications/Burp Suite Free Edition.app'
```
Comment (vitorgalvao) [inline Casks/burp-suite.rb]: It’s highly unlikely moving this is the only thing the installer does. Please try to figure out what other changes it performs.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/burp-suite.rb (hunk 1 above)
Comment (mcmahoniel) [inline Casks/burp-suite.rb]: Hi @vitorgalvao!

The installer only serves to build up the `.app` bundle in `/Applications`. The application is really just a wrapper for the standalone JAR.

I was able to confirm this with Instruments - the only file access is creating the `.app` and unpacking the files from the DMG.

(See also: `/Volumes/BurpSuiteFree/Burp Suite Free Edition Installer.app/Contents/Resources/app/`.)

File: Casks/burp-suite.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask 'burp-suite' do
+  version '1.7.06'
+  sha256 'ce2da473fdb65f4704ad6597dcd6615ec84e7a4c3c81deaf4f2de360d362a9bd'
+
+  url "https://portswigger.net/Burp/Releases/Download?productId=100&version=#{version.major_minor_patch}&type=MacOsx"
```
Comment (vitorgalvao) [inline Casks/burp-suite.rb]: Use just `#{version}`. `#{version.major_minor_patch}` doesn’t make sense in this case.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/burp-suite.rb (hunk 2 above)
Comment (mcmahoniel) [inline Casks/burp-suite.rb]: @vitorgalvao:

Fixed!

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**